[DOC] Restructure Pyroscope documentation and share content (#3798)
grafana/pyroscope: docs/sources/configure-server/about-ip-address-logging.md

  - /docs/phlare/latest/operators-guide/configuring/about-ip-address-logging/
  - /docs/pyroscope/latest/configure-server/about-ip-address-logging/
description: Troubleshoot errors by logging IP addresses of reverse proxies.
menuTitle: IP address logging of a reverse proxy
title: IP address logging of a reverse proxy
weight: 600
---

# IP address logging of a reverse proxy

> If a reverse proxy is used in front of Grafana Pyroscope, it might be difficult to troubleshoot errors.

Reviewer: I feel like we can make this sentence more active.

Author: ```suggestion
Note that using a reverse proxy in front of Grafana Pyroscope can make it difficult to troubleshoot errors.
```
